tests/persistence/test_trade_custom_data.py

```diff
@@ -85,8 +86,10 @@ def custom_exit(pair, trade, **kwargs):
     trade_after = Trade.get_trades_proxy(pair="ADA/USDT")[0]
     assert trade_after.get_custom_data("test_str") == "test_value"
     assert trade_after.get_custom_data("test_int") == 1
-    # 2 open pairs eligible for exit
-    assert ff_spy.call_count == 2
+    # 2 trades filled entry, with open exit order
+    # 1 trade with unfilled open entry order
+    # 1 trade with filled entry order
+    assert ff_spy.call_count == 4
```

Comment:
I'm not sure about the expected count being 4 here, as there is one trade that have not been filled we should expect 3 ?

![test_trade_custom_data_orders_debug](https://github.com/user-attachments/assets/47cf88f9-c0fa-4841-8d09-ac3e19c150b1)

Reply:
well the setup for test-trades wasn't correctly aligned for the "amount=0"-init change.

Fixing that and merging develop again, this now returns to 3 :D